js/interface.js

```diff
@@ -69,6 +73,10 @@ var definitionEditor = CodeMirror.fromTextArea($('#definition')[0], {
 
 var emptyColumnNameRegex = /^Column\s\([0-9]+\)$/;
 
+Fliplet.App.Tokens.get({appId: widgetData.appId, query: {type: 'integrationToken'}}).then(function(tokens) {
```

Comment:
I believe ESLint wants spaces for curly brackets `{ a, b }`

Reply:
did the changes